webanno-ui-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/annotation/detail/AnnotationFeatureForm.java

```diff
@@ -275,8 +248,7 @@ public void onSubmit(AjaxRequestTarget aTarget, Form<?> aForm)
             }
         });
 
-        add(layerSelector = new AnnotationFeatureForm.LayerSelector
-            ("defaultAnnotationLayer", annotationLayers));
+        add(layerSelector = new LayerSelector(this, "defaultAnnotationLayer", annotationLayers));
```

Comment:
Instead of directly passing the `annotationLayers` as a list, an IModel instance should be passed.